Packs/AWS/Integrations/AWS/AWS.py

```diff
@@ -1010,6 +1118,100 @@ def validate_args(args: Dict[str, Any]) -> dict:
             else:
                 raise e
 
+    @staticmethod
+    def describe_cluster_command(client: BotoClient, args: Dict[str, Any]) -> CommandResults:
+        """
+        Describes an Amazon EKS cluster.
+        Args:
+            client(boto3 client): The configured AWS session.
+            args: command arguments
+
+        Returns:
+            A Command Results object
+        """
+        cluster_name = args.get("cluster_name")
+
+        print_debug_logs(client, f"Describing clusters with parameters: {cluster_name}")
+        response = client.describe_cluster(name=cluster_name)
+        if response["ResponseMetadata"]["HTTPStatusCode"] != HTTPStatus.OK:
+            AWSErrorHandler.handle_response_error(response, args.get("account_id"))
+        response_data = response.get("cluster", {})
+        response_data["createdAt"] = datetime_to_string(response_data.get("createdAt"))
+        if response_data.get("connectorConfig", {}).get("activationExpiry"):
```

Comment:
Maybe extract the activation_expiry to a var and then use it. To clean this up a bit :)